go.mod

```diff
@@ -2,48 +2,46 @@ module github.com/red-hat-storage/ocs-client-operator
 
 go 1.18
 
-replace k8s.io/client-go => k8s.io/client-go v0.25.2
+replace (
+	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3 // required by rook v1.7
+	k8s.io/client-go => k8s.io/client-go v0.26.4
+	vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
```

Comment:
why do we need this one? i don't see this is required for ocs-operator as well.

Reply:
this is required for nooba-operator 5, If we do not use we will get this error while building.

`... go: downloading github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
go: github.com/openshift/cloud-credential-operator@v0.0.0-20210716145643-0ee1808fb8ed requires
    github.com/openshift/library-go@v0.0.0-20200911100307-610c6e9e90b8 requires
    vbom.ml/util@v0.0.0-20180919145318-efcd4e0f9787: unrecognized import path "vbom.ml/util": https fetch: Get "https://vbom.ml/util?go-get=1": dial tcp: lookup vbom.ml: no such host
ritesh:ocs-client-operator$`